python/nano/example/pytorch/quantization/inc/resnet18_cifar.py

```diff
@@ -92,6 +92,11 @@ def create_model():
     return model
 
 
+# Optional: replace loss and metrics
+criterion = torch.nn.NLLLoss()
+accuracy = Accuracy()
```

Comment:
why do you need this? we should minimize the changes

Reply:
The reason is mainly that we finally need to compile the qint8 model back to pl, while `trainer.compile(..)` accepts `torch.nn._Loss` and `torchmetrics` instead of `torch.functional`.  I was intended to keep both consistent.  
But, it is fine to fallback the changes here.

Reply:
> The reason is mainly that we finally need to compile the qint8 model back to pl, while `trainer.compile(..)` accepts `torch.nn._Loss` and `torchmetrics` instead of `torch.functional`. I was intended to keep both consistent. But, it is fine to fallback the changes here.

Why cannot `trainer.compile(..)` accept `torch.functional`?